groovy/gradle/external/binaries-list

```diff
@@ -0,0 +1,21 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+A271DEA739D4EBBD1A51F282AA130EB9314AD830 gradle-wrapper-4.10.2.jar
+70A74F970B22AC1226F6A2983E38A77FA90E3BE9 gradle-tooling-api-4.10.2.jar
```

Comment:
Isn't it better to download from Maven? Even if it would be non-central [repository](https://mvnrepository.com/artifact/org.gradle/gradle-tooling-api)?

Reply:
Will revisit later. mvn repository has no 4.10.2 which is required to work correctly with Java 11 and beyond.